brambling/tests/functional/test_order_views.py

```diff
@@ -87,3 +94,88 @@ def test_comped__sends_email(self):
         response = view.post(view.request)
         self.assertEqual(response.status_code, 302)
         self.assertEqual(len(mail.outbox), 2)
+
+
+class TransferViewTestCase(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+
+    def set_up_view(self, orderer=None, is_confirmed=False):
+        organization = OrganizationFactory(check_payment_allowed=True)
+        event_owner = PersonFactory()
+        OrganizationMember.objects.create(
+            person=event_owner,
+            organization=organization,
+            role=OrganizationMember.OWNER,
+        )
```

Comment:
Why do we need an event_owner? It doesn't _look_ like it's being used anywhere...